docs-guides/source/convert-a-torchvision-model-from-pytorch.md

```diff
@@ -122,21 +129,32 @@ Convert the model to an ML program using the Core ML Tools [`convert()`](https:/
 ```python
 # Using image_input in the inputs parameter:
 # Convert to Core ML using the Unified Conversion API.
-model = ct.convert(
+mlmodel_from_trace = ct.convert(
     traced_model,
     inputs=[image_input],
     classifier_config = ct.ClassifierConfig(class_labels),
     compute_units=ct.ComputeUnit.CPU_ONLY,
 )
 ```
 
+Torch.export path does not support image input yet, so it still uses tensor input
```

Comment:
---> Torch.export path does not support image input yet (as of coremltools 8.0)

Reply:
Added as suggested